contactPage/contactPage.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="styles.css" type="text/css" />
+    <title>Joseph:Home</title>
```

Comment:
Title should be above links to content. If you've linked to any third party css these should be above stuff you've made.